python/docs/intro.rst

```diff
@@ -0,0 +1,93 @@
+About the project
+=================
+
+Welcome to dlisio. dlisio is a python package for reading Digital Log
+Interchange Standard (DLIS) v1. A version 2 exist, and has been around for
+quite a while, but it is our understanding that most dlis files out there are
+still version 1. Hence dlisio's focus is put on version 1 [1]_, for now.
+
+dlisio is an attempt at abstract away a lot of the pain of dlis and give access
```

Comment:
I think 'attempt to abstract away ... and give' or 'attempt in abstracting away ... and giving'

Btw, my English is not the best, so it might be good if these header files are proofread by someone else.